HCore-Metadata/HCore-Metadata.csproj

```diff
@@ -7,12 +7,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="MaxMind.GeoIP2" Version="5.1.0" />
+    <PackageReference Include="MaxMind.GeoIP2" Version="5.2.0" />
     <PackageReference Include="Microsoft.Extensions.Caching.Redis" Version="2.2.0" />
-    <PackageReference Include="Nager.Country" Version="3.1.1" />
-    <PackageReference Include="Nager.Country.Translation" Version="3.0.3" />
+    <PackageReference Include="Nager.Country" Version="4.0.0" />
```

Comment:
Major version change. Tested?

Reply:
Yes, this works
It is used in ICountryMetadataProvider
When PortalsAPI-B initializes it registers ICountryMetadataProvider and both providers works